kratos/python_scripts/container_proxy.py

```diff
@@ -0,0 +1,65 @@
+__all__ = [
+    "ContainerProxy"
+]
+
+# --- Core Imports ---
+import KratosMultiphysics
+
+# --- STD Imports ---
+import typing
+
+
+##! @addtogroup KratosCore
+##! @{
+
+
+class ContainerProxy:
```

Comment:
I would get rid of this, since this does not make much sense in python. Users should directly call the respective functions rather than relying on this. And no computationally heavy operations should be done in python.

Reply:
done